langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -71,6 +89,17 @@ public isolated function iterator(table<MapType> t) returns object {
 # This for use in a case where it is known that the table has a specific key,
 # and accordingly panics if parameter `t` does not have a member with key parameter `k`.
 #
+# ```ballerina
+# table<record {|readonly string index; string name;|}> key(index) students = table [
+#         {index: "220001CS", name: "Jo"},
+#         {index: "220002CS", name: "Sam"}
+#     ];
```

Comment:
Same comment re: formatting.

Reply:
Please check wherever applicable.

Reply:
[#comment](https://github.com/ballerina-platform/ballerina-lang/pull/39084#discussion_r1054073900) I followed this formatting pattern.

Reply:
@MaryamZi